cephfs/cephfs_test.go

```diff
@@ -7,6 +7,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/ceph/go-ceph/rados"
+
```

Comment:
Do we (want to) follow a particular order of imports? If so, should any go-ceph import not be last?

Reply:
For the _test.go I liked putting the test-helpers external exports last, but I don't have particularly strong feelings about it.